### ISSUE TITLE
Some jenkins builds don't have links to the build logs for each pipeline

### ISSUE THREAD

**Issue by jhand2** [openenclave/openenclave#2878]
Sometimes when Jenkins runs a trying or staging build in the bors pipeline, it doesn't properly show links to the sub-pipelines that it ran.

This is what I see:

![image](https://user-images.githubusercontent.com/11011105/79479872-3eab0600-7fc2-11ea-8bee-78dd1ae36c02.png)

This is what I expect to see:

![image](https://user-images.githubusercontent.com/11011105/79479983-6306e280-7fc2-11ea-8150-745d7a17847e.png)

Note the "Triggered builds" section which links to the job/logs for that pipeline

**Comment by oprinmarius** [comment]
bors try-

**Comment by douglasmaciver-su** [comment]
I am working on writing this up.

**Comment by BRMcLaren** [comment]
Hello @jhand2 

There was a proposed fix for this but it does not seem to work 100% of the time, so let's also provide a workaround.

The bug in question seems to be one of Blue Ocean itself, as can be seen here: https://issues.jenkins-ci.org/browse/JENKINS-51111

1. If one looks at a job such as below and it did not contain a triggered jobs section, take note of the pipeline desired.

![image](https://user-images.githubusercontent.com/54328827/79670332-ef5a0680-8176-11ea-88d5-8057f0cd5230.png)

"Starting building: Pipelines » Azure-Linux #72" in this case.
2. Go to: https://oe-jenkins-dev.westeurope.cloudapp.azure.com/job/pipelines/ and into the pipeline that is failing.
![image](https://user-images.githubusercontent.com/54328827/79670394-7313f300-8177-11ea-8029-3fed04b420f8.png)


3, Burrow into the job number:

![image](https://user-images.githubusercontent.com/54328827/79670402-8626c300-8177-11ea-98e8-55bb05003e74.png)

4. Open Blue Ocean
![image](https://user-images.githubusercontent.com/54328827/79670436-e0278880-8177-11ea-8d64-1010ccc9e697.png)

5. View logs
![image](https://user-images.githubusercontent.com/54328827/79670444-f9303980-8177-11ea-9a56-2422a6d1b596.png)



TL:DR.

Agnostic-Linux : https://oe-jenkins-dev.westeurope.cloudapp.azure.com/blue/organizations/jenkins/pipelines%2FAgnostic-Linux/detail/Agnostic-Linux/<TheNumberNotedInStep1>/pipeline

Azure-Linux : https://oe-jenkins-dev.westeurope.cloudapp.azure.com/blue/organizations/jenkins/pipelines%2FAzure-Linux/detail/Azure-Linux/<TheNumberNotedInStep1> /pipeline

Azure- WIndows : https://oe-jenkins-dev.westeurope.cloudapp.azure.com/blue/organizations/jenkins/pipelines%2FAzure-Windows/detail/Azure-Windows/<TheNumberNotedInStep1>/pipeline

Please Notes that Bors# != Job Number

@jhand2 This should be left as an open bug for the meantime, in my opinion. At least until it is fully mitigated.

**Comment by AevaOnline** [comment]
notes from sig-testing meeting: this appears to be a known issue in Jenkins, and not likely to be fixed (upstream) any time soon.

**Comment by jhand2** [comment]
Link for open Jenkins/Blue-Ocean issue: https://issues.jenkins-ci.org/browse/JENKINS-60995?jql=project%20%3D%20JENKINS%20AND%20status%20in%20(Open%2C%20%22In%20Progress%22%2C%20Reopened)%20AND%20component%20%3D%20blueocean-plugin

**Comment by jhand2** [comment]
Since we upgraded from BlueOcean 1.22 to 1.23, I have not seen this issue. I will keep looking out for it. If I don't see a repro by next triage meeting I will close this issue.

**Comment by jhand2** [comment]
Just kidding, just saw it again https://oe-jenkins-dev.westeurope.cloudapp.azure.com/blue/organizations/jenkins/Bors/detail/trying/362/pipeline

:frowning:

**Comment by BRMcLaren** [comment]
I recently found a faster way to get around this bug, wanted to share. If you go to your Bors logs:

![image](https://user-images.githubusercontent.com/54328827/91083954-09fa2b80-e600-11ea-9223-b63b55f99c38.png)

Click the classic UI button.

![image](https://user-images.githubusercontent.com/54328827/91084021-2e560800-e600-11ea-92b1-db16d985f640.png)

Then if you go to the job, it will show the unstable build links.

![image](https://user-images.githubusercontent.com/54328827/91084083-462d8c00-e600-11ea-8bf9-c2bd51d74f6e.png)

and you have logs!

![image](https://user-images.githubusercontent.com/54328827/91084145-5d6c7980-e600-11ea-86cb-f28d0eec007f.png)

**Comment by sipani909** [comment]
I am still seeing this on 2.391 and blue ocean 1.2.7.1
